docs/4.4/enterprise/ssh-rbac.md

```diff
@@ -272,11 +271,11 @@ spec:
       'environment': ['test', 'staging']
       # regular expressions are also supported, for example the equivalent
       # of the list example above can be expressed as:
-      'environment': "{% raw %}{{regexp.match("^test|staging$")}}{% endraw %}"
+      'environment': '{% raw %}{{regexp.match("^test|staging$")}}{% endraw %}'
```

Comment:
These two should match. We can't use one formatting for one and then invert it 4 lines below.

Reply:
We should also actually test these formats and put the one that works in the docs, because YAML quoted strings can be weird.

Reply:
Looks like this was fixed when I merged upstream